src/styles/product_card.scss

```diff
@@ -0,0 +1,112 @@
+.product {
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition: all 300ms;
```

Comment:
The `transition: all 300ms;` rule can lead to performance issues if there are many properties that could potentially be animated, which is not always necessary. It's better to specify which properties you want to transition. For example, if you want to transition the `transform` property, you should write `transition: transform 300ms;` instead.